pkg/database/realm.go

```diff
@@ -434,6 +429,33 @@ func (r *Realm) BeforeSave(tx *gorm.DB) error {
 	return nil
 }
 
+// validateSMSTemplate is a helper method to validate a single SMSTemplate.
+// Errors are returned by appending them to the realm's Errorable fields.
+func (r *Realm) validateSMSTemplate(t string) {
+	if r.EnableENExpress {
+		if !strings.Contains(t, SMSENExpressLink) {
+			r.AddError("SMSTextTemplate", fmt.Sprintf("must contain %q", SMSENExpressLink))
```

Comment:
these errors aren't going to be right, since they are for SMSTextTemplate

validation is going to be tricky...

this is where I think breaking these out into a separate table so you had a stable ID that could be part of a form field ID would be valuable.

Reply:
We should Take the field name as another param to this function. We could also prefix the error with the label name for alt templates.

That being said, we'll mark these as `required` in the HTML, so the only way someone gets to these errors is if they do some significant mucking about. I think it's probably fine to keep this as-is and create a follow-up issue to make it betterer in the future. WDYT?

Reply:
we can also include the label in the error message - I'm worried about a user not being able to tell where the error is
and therefore not be able to fix it.

Reply:
I'm fine w/ doing the follow up when the UI is added.